tests/pkg_wolfcrypt-ed25519-verify/tools/ed25519_sign_msg.c

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2019 Daniele Lacamera
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+#include <stdint.h>
+
+#include <wolfssl/wolfcrypt/settings.h>
+#include <wolfssl/wolfcrypt/ed25519.h>
+
+void print_buf(uint8_t *buf, int len)
```

Comment:
(Again, comment only): You could use module `ocd` here to safe some lines of code here

Reply:
agree, done